sci-astronomy/stellarium/files/stellarium-0.24.3-no-tbb.patch

```diff
@@ -0,0 +1,12 @@
+Don't depend on libtbb
+
+--- a/cmake/Check-std_execution.cmake
++++ b/cmake/Check-std_execution.cmake
+@@ -3,6 +3,7 @@ try_compile(STD_EXECUTION_KNOWN "${CMAKE_BINARY_DIR}" "${CMAKE_CURRENT_LIST_DIR}
+ if(STD_EXECUTION_KNOWN)
+ 	message(STATUS "Checking that std::execution::par is supported by the C++ library - yes")
+ else()
++	message(FATAL_ERROR "Gentoo: std::execution::par not available")
```

Comment:
Ok, I found this now. :)

This seems to assume that all compilers can use `std::execution::par` without tbb. However, this does not seem to be the case, as I run into this fatal error on a mostly stable system.

Not sure what the best way forward is. Options include:
1. Globally disable the usage of std::execution (until all compilers support it without tbb)
2. Uncondtionally depend on tbb
3. Only enable `std::execute` if it is supported without tbb

Option three strikes me as the best middle ground. So instead of adding this fatal error here, you could simply drop the else case.

Reply:
> Only enable std::execute if it is supported without tbb

Any idea how to achieve this? If we just drop the fatal error, then the feature will be randomly enabled or not depending on whether tbb happens to be installed at the time. The guidelines for writing ebuilds told to avoid such automagic. And perhaps it will stop working if the library is uninstalled later?

Reply:
Considering gcc also needs tbb (at least in some versions), I went with unconditional tbb for now.